### PR TITLE
Fix anon bug

### PR DESCRIPTION
**Pull request by kazaamjt**
Needed if app is deployed on same machine as the database.

### CODE REVIEW

File: model/_init.cf [hunk 1]
```diff
@@ -40,6 +40,7 @@ entity Server extends ip::services::Server, DBMS:
     """
         Mysql server configuration
     """
+    bool remove_anon_users = false
```
Comment (arnaudsjs) [inline model/_init.cf]: Maybe add a docstring to clarify what this attribute does.

File: model/_init.cf [hunk 2]
```diff
@@ -68,6 +69,12 @@ end
 
 implement ManagedMysql using manageManaged
 
+implementation removeAnonUsers for Server:
+    exec::Run(host=host, command="/usr/bin/mysql -s -e \"delete from user where User = ''; flush privileges;\" mysql")
```
Comment (arnaudsjs) [inline model/_init.cf]: * The exec::Run requires the mysql service. 
* The exec::Run should provide Server

File: model/_init.cf (hunk 1 above)
Comment (kazaamjt) [reply model/_init.cf]: resolved

File: model/_init.cf (hunk 2 above)
Comment (kazaamjt) [reply model/_init.cf]: resolved

Comment (kazaamjt) [reply model/_init.cf]: resolved

File: model/_init.cf [hunk 3]
```diff
@@ -39,9 +39,15 @@ ManagedMysql.agenthost [1] -- ip::Host
...
     """
         Mysql server configuration
+
+        :attr remove_anon_users: Required when trying to connect to a mysql database on the same host over it' external IP.
     """
+    bool remove_anon_users = false
 end
 
+# svc is required for remove_anon_users
+Server.svc [1] -- std::Service
```
Comment (arnaudsjs) [inline model/_init.cf]: Let's change Server.svc to Server._svc to make clear that _svc is set/used internally and should not be set by the end-user of the module.

Comment (kazaamjt) [reply model/_init.cf]: resolved

**Pull request closed without merging**